eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1492,6 +1492,14 @@ protected function internalPublishVersion(APIVersionInfo $versionInfo, $publicat
         return $content;
     }
 
+    /**
+     * @return int
+     */
+    protected function getTime()
```

Comment:
Let's be more specific - `getUnixTimestamp`
also you could strict return value

Reply:
about strict return value I double checked and in this class, we not have it

Reply:
method name fixed @alongosz